autoload/go/debug.vim

```diff
@@ -25,6 +25,16 @@ if !exists('s:start_args')
   let s:start_args = []
 endif
 
+if !exists('s:default_debug_mappings')
+  let s:default_debug_mappings = [
+    \["nmap", "<F5>",  "<Plug>(go-debug-continue)"],
+    \["nmap", "<F6>",  "<Plug>(go-debug-print)"],
+    \["nmap", "<F9>",  "<Plug>(go-debug-breakpoint)"],
+    \["nmap", "<F10>", "<Plug>(go-debug-next)"],
+    \["nmap", "<F11>", "<Plug>(go-debug-step)"],
```

Comment:
This slightly differs from the default. I tested it with <buffer> but when I step into some file the buffer changes and the mappings are no longer active, thus I decided to leave it without the buffer attribute

Reply:
Ideally the behavior should be that these mappings will be only active in a certain window or tab. E.g. I have the n key mapped to step over in debug mode and I usually have the n key mapped to next search result, I would like the semantics of n to be the default in other tabs. WDYT?